scripts/zones/Garlaige_Citadel/MobIDs.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Area: Garlaige Citadel
+-- Comments: -- posX, posY, posZ
+--(Taken from 'mob_spawn_points' table)
+-----------------------------------
+
+-- Hovering Hotspot
+Hovering_Hotpot=17596628;
+Hovering_Hotpot_PH={ 
```

Comment:
Brace on newlines please. I know the project had a lot of old files done that way, I just now mass corrected them.